spec/controllers/api/v0x0/tasks_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe Api::V0x0::TasksController, :type => :request do
+  it("Uses IndexMixin")   { expect(described_class.instance_method(:index).owner).to eq(Api::V0::Mixins::IndexMixin) }
+  it("Uses ShowMixin")    { expect(described_class.instance_method(:show).owner).to eq(Api::V0::Mixins::ShowMixin) }
+  it("Uses UpdateMixin")  { expect(described_class.instance_method(:update).owner).to eq(Api::V0::Mixins::UpdateMixin) }
+
+  let(:tenant) { Tenant.create! }
+
+  it "patch /tasks/:id updates a Task" do
```

Comment:
Is this test necessary?  It should be covered by the UpdateMixin spec

Reply:
Doesn't that spec just test that it includes the `#update` method? This spec is making sure that if you pass in the status and context those get updated. I'd argue we could also additionally pass in another parameter that we expect *not* to get updated, like name or something.